src/main/msp/msp.c

```diff
@@ -1808,7 +1808,7 @@ case MSP_NAME:
     case MSP_RC_DEADBAND:
         sbufWriteU8(dst, rcControlsConfig()->deadband);
         sbufWriteU8(dst, rcControlsConfig()->yaw_deadband);
-        sbufWriteU8(dst, rcControlsConfig()->alt_hold_deadband);
+        sbufWriteU8(dst, rcControlsConfig()->pos_hold_deadband);
```

Comment:
:eyes:

Reply:
Yes... naughty of me ;-)

However the Position Hold deadband seems more useful, now we have it, than Altitude Hold Deadband.  And we ha ve only one existing MPS slot available :-). 

Have to make a final decision on this location but it seems more useful for Position Hold.

Reply:
Moved both these deadbands to their respective config / pg files.  Better like that.  Thanks.